add-dynamic-entity-layer/src/main/java/com/esri/arcgismaps/sample/adddynamicentitylayer/components/BottomSheetContent.kt

```diff
@@ -0,0 +1,182 @@
+                color = MaterialTheme.colorScheme.primary
+            )
+            TextButton(
+                onClick = onDismiss
+            ) {
+                Text(text = "Done")
+            }
+        }
+
+        Surface(
+            modifier = Modifier.padding(20.dp),
+            tonalElevation = 1.dp,
+            shape = RoundedCornerShape(20.dp),
+            border = BorderStroke(1.dp, MaterialTheme.colorScheme.outlineVariant)
+        ) {
+            Column(
+                modifier = Modifier.padding(14.dp)
+            ) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    Text(
+                        text = "Track Lines",
+                        style = SampleTypography.bodyLarge
+                    )
+                    Switch(
+                        checked = isTrackLineVisible,
+                        onCheckedChange = {
+                            onTrackLineVisibilityChanged(it)
+                        }
+                    )
+                }
+                Divider(thickness = 0.5.dp)
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+
+                    Text(
+                        text = "Previous Observations",
+                        style = SampleTypography.bodyLarge
+                    )
+                    Switch(
+                        checked = isPrevObservationsVisible,
+                        onCheckedChange = {
+                            onPrevObservationsVisibilityChanged(it)
+                        }
+                    )
+                }
+            }
+        }
+
+        Surface(
+            modifier = Modifier.padding(20.dp),
+            tonalElevation = 1.dp,
+            shape = RoundedCornerShape(20.dp),
+            border = BorderStroke(1.dp, MaterialTheme.colorScheme.outlineVariant)
+        ) {
+            Column(
+                modifier = Modifier.padding(14.dp)
+            ) {
+                Column {
+                    Row {
+                        Text(
+                            modifier = Modifier.weight(12f),
+                            text = "Observations per track",
+                        )
+                        Text(
+                            modifier = Modifier.weight(1f),
+                            text = sliderValue.toInt().toString()
+                        )
+                    }
+                    Slider(
+                        value = sliderValue,
+                        onValueChange = {
+                            sliderValue = it
+                            onObservationsChanged(sliderValue)
```

Comment:
And you could pass in the `observationsPerTrack` to the slider and Compose would handle observation changes
```suggestion
                            text = observationsPerTrack.roundToInt().toString()
                        )
                    }
                    Slider(
                        value = observationsPerTrack,
                        onValueChange = {
                            onObservationsChanged(it)
```